constants/error-constants.js

```diff
@@ -222,7 +222,9 @@ const ERROR = {
     INVALID_USER_SCOPE: "Invalid user scope permission is requested",
     INVALID_SCOPE_VALUES: "You have the required permission $item$, but no values are assigned to it.",
     // Maintenance Mode
-    MAINTENANCE_MODE: "The CRDC Submission Portal site is currently undergoing scheduled maintenance. Please check back soon. We appreciate your patience."
+    MAINTENANCE_MODE: "The CRDC Submission Portal site is currently undergoing scheduled maintenance. Please check back soon. We appreciate your patience.",
+    INVALID_PARTICIPANT_NOT_FOUND: "The specified participant was not found.",
```

Comment:
If these two constants are needed, please remove "INVALID_" part from the names, it's hard to read and understand.